### ISSUE TITLE
Do not merge specifications into website

### ISSUE THREAD

**Issue by effigies** [bids-standard/bids-website#432]
I hadn't realized that spun-off specs were part of the websites might be consolidated. The website IMO is for public communication, so the starter kit for instance makes sense. Specifications that are (or should be) versioned and provide (or should provide) dedicated tooling seem like poor fits to me. For example, stats-models was written as a JupyterBook along with a Pydantic model that could generate JSON schema. That seems like a very poor fit for an mkdocs site.

I would suggest that BEPs 2 (Stats Models), 27 (Execution) and 28 (Provenance) all continue to be separate sites. Although the maintenance costs of more repositories can be high, I think the costs of integration are higher, particularly with regard to clarity about whether something is spec or convention.

I would be happy to consider methods for improving the maintainability of these sites by settling on common tooling and making upgrading all of them simultaneously simpler.

_Originally posted by @effigies in https://github.com/bids-standard/bids-website/issues/430#issuecomment-2169750941_

**Comment by Remi-Gau** [comment]
Yeah agree that those ones are have some content that should probably stay there, but both stats and execution have content that is not just pure spec either (examples, tutorials) so some content could also probably be moved out of them.

The move to the new website will take a bit of time: I may have jumped the gun a a bit by moving the issues.

**Comment by effigies** [comment]
Agreed, and I think we should do a better job of separating spec from other documentation along the lines of https://diataxis.fr. I think a lot of that still fits naturally inside the spec, especially if you use JupyterBook to demonstrate your tutorials and how-tos using the spec-related tooling, but I'm okay with some stuff relating to these specs sitting outside the specs.

**Comment by Remi-Gau** [comment]
FYI: I will probably move [this page](https://bids-standard.github.io/stats-models/json_101.html) out of the stats-model as it seems that it could benefit to be in a more generic place.

**Comment by effigies** [comment]
Zero problem with that. I think common tutorials would be great to consolidate and then link to from all of our different specs, and we can reduce duplication that way.

A crash course in data formats would be a good section. We could adapt some nipy text for NIfTI. Might even be useful with just NIfTI and JSON, given that's still the majority of BIDS, but it would be nice to aspire to cover all supported formats.

**Comment by Remi-Gau** [comment]
BTW I think that if people feel that JupyterBook tutorials should be part of our main doc, then I think we could possibly host the new BIDS website in a jupyter book.

**Comment by Remi-Gau** [comment]
Moving this to an actual discussion as this where giscus comments will go when people comment on the new website.
